RSS Reader/RSS Reader/Categories/NSError/NSError+UserFriendlyError.h

```diff
@@ -0,0 +1,18 @@
+//
+//  NSError+UserFriendlyError.h
+//  RSS Reader
+//
+//  Created by Lina Loyko on 12/7/20.
+//
+
+#import <Foundation/Foundation.h>
+
+NS_ASSUME_NONNULL_BEGIN
+
+@interface NSError (UserFriendlyError)
+
+- (void)userFriendlyErrorWithCompletion:(void (^)(NSString * _Nonnull title, NSString * _Nonnull message)) completion;
```

Comment:
Just a recommendation. Naming approach should provide meaningful though short names. A better name would be something like -parseError

Reply:
Done